### DISCUSSION TITLE
2.4 Beta 502 status code

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#10089] [2.4]
I am currently experiencing a "status code 502". I had this code pop up once before and in order to get rid of it I deleted Elastic Data and restarted the instance.

The 502 code came back once I tried to push an event to cases. Any suggestions?

![image](https://user-images.githubusercontent.com/70167373/229878361-f804dfb3-d904-4a06-ad61-c9aa0c7bf335.png)

**Comment by TOoSmOotH** [comment]
What does the sensoroni.log say?

**Reply by ColeVan** [reply]
Looking through the log everything looks good now. A simple reboot may have fixed the problem.
![image](https://user-images.githubusercontent.com/70167373/230141608-f8217cf3-d7b5-45a1-b956-8c995ba633dc.png)
